Objects/longobject.c

```diff
@@ -4312,7 +4413,7 @@ long_pow(PyObject *v, PyObject *w, PyObject *x)
            We could _always_ do this reduction, but l_divmod() isn't cheap,
```

Comment:
Looks like the comment above is rendered obsolete, since `l_divmod()` is no longer called. Should be changed to say `l_mod()` instead, right?

Reply:
Fixed.